README.md

```diff
@@ -199,7 +227,6 @@ DP+ tables will be created with the command:
 ckan -c /etc/ckan/default/ckan.ini datapushers db-init
```

Comment:
@tino097 How is the proper way to initialize the datapusher database?

I'm having issues because running `ckan db migrate -p datapusher_plus` throws an error: `sqlalchemy.exc.NoSuchTableError: jobs`

The problem is that the first migration assumes there is a table called jobs that doesn exist.

Trying to run 'ckan datapusher_plus db_init` does not work (Not such command `datapusher_plus`). datapusher_plus db-init`.